src/json_stream/base.py

```diff
@@ -106,6 +106,10 @@ def __len__(self) -> int:
     def __repr__(self):  # pragma: no cover
         return f"<{type(self).__name__}: {repr(self._data)}, {'STREAMING' if self.streaming else 'DONE'}>"
 
+    def __contains__(self, item):
+        self.read_all()
```

Comment:
This could be implemented so that, if `item` is near the start, you don't have to read the whole thing.